package exceptiondemo2;

public class NullPointerExceptionDemo {
	
	
	static NullPointerExceptionDemo obj2;
	
	static String str;
	
	static String str1="";
	
	static String str2=null;
	

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
//		
//		NullPointerExceptionDemo obj = new NullPointerExceptionDemo();
//		
//		NullPointerExceptionDemo obj1 = new NullPointerExceptionDemo();
//		
//		try {
//			System.out.println(obj.equals(obj1));
//			
//			System.out.println(obj2.equals(obj1));
//		} 
//		
//		catch (Exception e) {
//			// TODO Auto-generated catch block
//			e.printStackTrace();
//			
////			System.out.println(e.getMessage());
//		}
//		
//		
//		System.out.println("Hello");
//		
//		
//		System.out.println("end of program");
		
//		System.out.println(str.length());
//		System.out.println(str1.length());
//		System.out.println(str2.length());
		
		
		

	}

}
